backends-velox/src/main/scala/org/apache/gluten/execution/ColumnarCollectLimitExec.scala

```diff
@@ -125,11 +199,21 @@ case class ColumnarCollectLimitExec(
       if (childRDD.getNumPartitions == 1) childRDD
       else shuffleLimitedPartitions(childRDD)
 
-    processedRDD.mapPartitions(partition => collectLimitedRows(partition, limit))
+    processedRDD.mapPartitions(
+      partition => {
+        val droppedRows = dropLimitedRows(partition, offset)
```

Comment:
We can add the argument `offset` to `collectLimitedRows`, just change it in function `fetchNext`, it can make the function much easier, right?